students/wachulec_maria/lesson_9_reading_and_writing_files/regex_search.py

```diff
@@ -0,0 +1,26 @@
+import os
+import re
+
+
+def search_in_file(file, reg):
+    txt_file = open(os.path.join(os.getcwd(), file))
+    text = txt_file.read()
+    regex = (re.findall(reg, text))
```

Comment:
what is the use of extra parenthesis in line 8 ?

Reply:
I forgot to remove this parenthesis